README.md

```diff
@@ -0,0 +1,35 @@
+README:
+
+Team mike's UML Editor
+
+Team mike's UML Editor is an application that allows the user to add, delete, and edit UML
+classes. Each UML class can hold different fields and methods, which can be added, deleted,
+and renamed. The user can also save files and load other ones to easily save progress and 
+continue to work on it at a later time. The application currently runs only using a Command
+Line Interface.
+
+Requirements to use the application:
```

Comment:
Since this is a smaller subsection, we should stick to less "#". Maybe only two?